src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java

```diff
@@ -296,13 +304,26 @@ void processPresence(Presence presence)
             statsId = statsIdPacketExt.getStatsId();
         }
 
+        boolean wasAudioMuted = isAudioMuted;
+        StandardExtensionElement audioMutedExt = presence.getExtension("audiomuted", "jabber:client");
+        isAudioMuted = audioMutedExt == null || Boolean.valueOf(audioMutedExt.getText()); /* defaults to true */
+
+        if (isAudioMuted != wasAudioMuted)
+        {
+            logger.debug(() -> toString() + ". isAudioMuted = " + isAudioMuted + ".");
```

Comment:
If you need extra context for the `ChatMemberImpl` instance in the logs you can add it to the logger, e.g.
`logger.addContext("occupantJid", occupantJid.toString());`. We could also hook it up to the "parent" logger so it has context for the chat room/conference.

Reply:
added occupantJid to the logging context.